### DISCUSSION TITLE
[Reset] Display 'initial' value

### DISCUSSION THREAD

**Discussion by Toub** [tailwindlabs/tailwindcss#4623] [Ideas]
What about adding a responsive "display: initial" attribute.

Would be very usefull when hidding an item without mind about its initial value.

Example:

Current:
```html
<td class="hidden md:table-cell">Visible only if screen >= MD</td>
```

Proposal:
```html
<td class="hidden md:display-initial">Visible only if screen >= MD</td>
```

Even better if it could be set for every attribute: https://www.w3.org/TR/css-cascade-3/#initial

**Comment by quantuminformation** [comment]
yeh we need this

**Comment by Carlsia** [comment]
Any course of action on this, I'm having a relative issue. I tried to change things and this is what happened.
[myBayCare](https://www.baycare-patientportal.com/)
